config/sudoers

```diff
@@ -19,6 +19,12 @@ Defaults	secure_path="/vx/code/config/vendor-functions:/usr/local/sbin:/usr/loca
 # User privilege specification
 root	ALL=(ALL:ALL) ALL
 
+# Defaults for all users per command
+# !log_allowed disables logging allowed commands but logs failures
+# !pam_session disables a pam session for the command
```

Comment:
Just making sure I understand: A pam session isn't needed for our calls to the commands below, correct? My mistaken understanding was that we needed a pam session but just wanted to stop the log. What I'm seeing here is that the pam session is entirely unnecessary

Reply:
That's correct.